core/lib/types/src/event.rs

```diff
@@ -51,6 +53,61 @@ static L1_MESSAGE_EVENT_SIGNATURE: Lazy<H256> = Lazy::new(|| {
     )
 });
 
+// struct L2ToL1Log {
```

Comment:
Would it make sense to add a bit more context to this comment? E.g. prepend with "Solidity structure counterpart:" and move the structure into a markdown code block.